src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py

```diff
@@ -524,16 +524,33 @@ def check_inputs(
                     f" {negative_prompt_embeds.shape}."
                 )
 
-    def get_timesteps(self, num_inference_steps, strength, device):
+    def get_timesteps(self, num_inference_steps, strength, device, denoising_start=None):
```

Comment:
Can we add a `#copy from` statement here - I think here we can use same `get_timesteps` function in SDXL img2img, no?

note that the SDXL img2img `get_timesteps` implementation include code to make sure mixure-of-exports work correctly with 2nd order scheduler (see PR here https://github.com/huggingface/diffusers/pull/5511)
```suggestion
     # Copied from diffusers.pipelines.stable_diffusion_xl.pipeline_stable_diffusion_xl.StableDiffusionXLImg2ImgPipeline.get_timesteps
    def get_timesteps(self, num_inference_steps, strength, device, denoising_start=None):
```